packages/server/lib/browsers/electron.js

```diff
@@ -263,10 +280,7 @@ module.exports = {
   _enableDebugger (webContents) {
     debug('debugger: enable Console and Network')
 
-    return Bluebird.join(
-      webContents.debugger.sendCommand('Console.enable'),
-      webContents.debugger.sendCommand('Network.enable'),
```

Comment:
this ended up not being necessary, and was probably causing a lot of unnecessary background traffic over the debugger :grimacing:

Reply:
interesting